### ISSUE TITLE
"Error: Parser: 1:0: Unexpected < when seeking value" when declining friend request

### ISSUE THREAD

**Issue by Hello71** [bitlbee/bitlbee-steam#121]
```
<@root> steam - New request: The user [snip] ([snip]) wants to add you to his/her buddy list.
<@root> You can use the yes/no commands to accept/reject this request.
<@Hello71> no
<@root> steam - Rejected: The user [snip] ([snip]) wants to add you to his/her buddy list.
<@root> steam - Error: Parser: 1:0: Unexpected < when seeking value
<@root> steam - Signing off..
```

**Comment by jgeboski** [comment]
What version of the plugin are you using?

**Comment by Hello71** [comment]
1.4.1: https://bugs.gentoo.org/show_bug.cgi?id=580786

**Comment by jgeboski** [comment]
This has been resolved as of 7cc92aa.
